src/parseImage.py

```diff
@@ -803,6 +809,10 @@ def handlePARROT(xmp_str, exifData):
     else:
         # # debug printout
         # print(f'y: "{y}" x: "{x}" z: "{z}" azimuth: "{azimuth}" theta: "{theta}"')
+
+        #if model name contains "anafiai" parrot drone uses EGM96 and needs conversion
+        if "anafiai" in xmp_str.lower():
```

Comment:
@SlicedBacon Logic error, this conditional should be `not in` instead of `in`

The Anafi AI is Parrot's only drone that uses WGS84 for vertical datum. All others are EGM96

See:
https://github.com/Theta-Limited/OpenAthenaAndroid/blob/17edef7234894ed4a9342032aded1498e87183f5/app/src/main/java/com/openathena/MetadataExtractor.java#L514
https://developer.parrot.com/docs/groundsdk-tools/photo-metadata.html

Reply:
Code comment should also be updated to match

Reply:
Oops. Fixed.